### PR TITLE
fix: card responsive in drawer interview details

### PR DESCRIPTION
**Pull request by Naiby**

### CODE REVIEW

File: src/common/types/User.ts [hunk 1]
```diff
@@ -8,6 +8,7 @@ export type User = {
   email: string
   firstName: string
   lastName: string
+  phone: number
```
Comment (juliswer) [inline src/common/types/User.ts:11]: We are not waiting for the phone number on the Interviewers or Recruiters!

File: src/components/InterviewDetail/InterviewDrawer.tsx [hunk 2]
```diff
@@ -106,6 +93,8 @@ export function InterviewDrawer() {
     if (!isOpenDrawer) setZoomCopied(false)
   }, [zoomCopied, isOpenDrawer])
 
+  console.log(interview)
```
Comment (juliswer) [inline src/components/InterviewDetail/InterviewDrawer.tsx:96]: remove clg

File: src/components/InterviewDetail/InterviewDrawer.tsx [hunk 3]
```diff
@@ -117,7 +106,8 @@ export function InterviewDrawer() {
       <SpinnerContainer isLoading={isLoadingDrawer}>
         <div className="p-5">
           <h1 className="mt-10 text-2xl font-extrabold text-center">
-            Interview with {candidateName}
+            Interview with
```
Comment (juliswer) [inline src/components/InterviewDetail/InterviewDrawer.tsx:109]: Why did you delete the utils in this instead of moving them to a single file to be reused?

File: src/components/InterviewDetail/InterviewDrawer.tsx [hunk 4]
```diff
@@ -196,31 +186,12 @@ export function InterviewDrawer() {
...
-                        </div>
-                      </div>
-                    </div>
+                  <div className="w-full h-full">
+                    <CardProfile profile={interview.candidate} />
                   </div>
-                  <div className="relative w-full h-full lg:w-3/5 lg:right-column">
-                    {/* Uncomment on the 2nd sprint  */}
-                    {/* <button className="gap-1 btn" type="button">
+                  {/* <div className="relative w-full h-full lg:w-3/5 lg:right-column"> */}
```
Comment (juliswer) [inline src/components/InterviewDetail/InterviewDrawer.tsx:192]: remove commented code

File: src/components/common/CardProfile.tsx [hunk 5]
```diff
@@ -0,0 +1,45 @@
+import { type Candidate } from '@/common/types/Candidate'
+import { type Interviewer } from '@/common/types/Interviewer'
+import { getAvatarName } from '@/common/utils/get-avatar-name'
+import clsx from 'clsx'
+import { Avatar } from './Avatar'
+/* eslint-disable react/require-default-props */
```
Comment (juliswer) [inline src/components/common/CardProfile.tsx:6]: small comment. Please move this rule to the top of the file

**Review by juliswer — CHANGES_REQUESTED** [review]
Please check my comments

File: src/common/types/User.ts (hunk 1 above)
Comment (AlefrankM) [reply src/common/types/User.ts:11]: Yeah, but I think that we can use this field for everyone because in the future we can add this field

File: src/components/InterviewDetail/InterviewDrawer.tsx (hunk 3 above)
Comment (Naiby) [reply src/components/InterviewDetail/InterviewDrawer.tsx:109]: Yep

File: src/components/InterviewDetail/InterviewDrawer.tsx (hunk 4 above)
Comment (Naiby) [reply src/components/InterviewDetail/InterviewDrawer.tsx:192]: But they are buttons that were commented for their functionalities in this sprint 2. no? No comments from me.

Comment (AlefrankM) [reply src/components/InterviewDetail/InterviewDrawer.tsx:192]: This code will be used!

**Review by angelstackit — APPROVED** [review]

**Review by AlefrankM — APPROVED** [review]

**Review by AlefrankM — APPROVED** [review]
LGTM

**Review by angelstackit — APPROVED** [review]
hmm...
